atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -170,6 +172,24 @@ def validate_platforms_in_base_image(self, base_image):
                           'because platform descriptors are not defined')
             return
 
+        if '@sha256:' in str(base_image):
+            base_image_result = ImageName.parse(str(base_image))
+            base_image = base_image_result.copy()
+
+            try:
+                config_blob = get_config_from_registry(base_image, base_image.registry,
+                                                       base_image.tag,
+                                                       insecure=self.parent_registry_insecure)
+            except (HTTPError, RetryError, Timeout) as ex:
+                self.log.warning('Unable to fetch config for %s, got error %s',
+                                 base_image, ex.response.status_code)
+                raise RuntimeError('Unable to fetch config for base image')
```

Comment:
Why are you raising an error here, instead of falling through to the get_manifest_list() path?
what's the advantage of searching for the manifest_list by the manifest_list_name instead of the v2 sha digest?

Reply:
and we can't search manifest list with v2 sha digest only